.gitmodules

```diff
@@ -6,3 +6,18 @@
     path = dependencies/imgui_filedialog
     url = https://github.com/aiekick/ImGuiFileDialog.git
     branch = "Lib_Only"
+[submodule "dependencies/glm"]
+    path = dependencies/glm
+    url = https://github.com/g-truc/glm.git
+[submodule "dependencies/sdl"]
+    path = dependencies/sdl
+    url = https://github.com/libsdl-org/SDL.git
+[submodule "dependencies/glbinding"]
+    path = dependencies/glbinding
+    url = https://github.com/cginternals/glbinding.git
+[submodule "dependencies/plog"]
+    path = dependencies/plog
+    url = https://github.com/SergiusTheBest/plog.git
+[submodule "dependencies/assimp"]
+    path = dependencies/assimp
+    url = https://github.com/assimp/assimp.git
```

Comment:
you won't see the `branch = ` fields here, because I literally went through all the submodules dirs and switched them to certain **TAG**
this may not be the best approach, but it works